### DISCUSSION TITLE
CLTV36

### DISCUSSION THREAD

**Discussion by youngbluespring** [iptv-org/community#869] [Q&A]
### Channel ID

CLTV36.ph

### Stream URL

https://cltv36.tv/

### Notes

The live stream is embedded in the homepage

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by fotvnetwork** [comment]
https://live.cltv36.tv:5443/LiveApp/streams/live_adaptive.m3u8?token=undefined&subscriberId=undefined&subscriberCode=undefined

**Comment by freearhey** [comment]
https://live.cltv36.tv:5443/LiveApp/streams/live_adaptive.m3u8?token=undefined&subscriberId=undefined&subscriberCode=undefined (offline)
<img width="828" alt="image" src="https://user-images.githubusercontent.com/7253922/232257844-d1e022b6-8e0e-44c7-838f-e81592fb5dd2.png">

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/ph.m3u#L12-L13
